app/helpers/wpcc/your_results_page_helper.rb

```diff
@@ -0,0 +1,9 @@
+module Wpcc
+  module YourResultsPageHelper
+    TAX_RELIEF_THRESHOLD_RATE = 11_500
+
+    def earning_below_tax_relief_threshold?(salary)
+      salary.to_i < TAX_RELIEF_THRESHOLD_RATE
+    end
+  end
+end
```

Comment:
As I commented above. I agree with only one pattern. Emphasizing again, this PR helper is very good, *but* we should use presenters. Presenters (or decorators) are usually composition objects that add presentation logic to attributes. 
Perhaps it is good to have a DLT around it too. https://martinfowler.com/eaaDev/PresentationModel.html
We only need to define about which presenter will be the responsible to handle this rule. @Tr4pSt3R feel free tomorrow to pair with any of us to solve this salary attribute.